book/website/collaboration/github/github-roadmapping.md

```diff
@@ -1,10 +1,10 @@
-(rr-github-roadmapping)=
+(cl-github-roadmapping)=
 # Roadmapping
 
-If you plan a larger piece of work, it is very useful to have an outline for the work you need to do and share it with potential contributors.
-Your roadmap covers your goal and vision and should include a timeline for tasks that need to be completed, thus helping anyone new to your project to develop an understanding of what is currently happening on the project and what's coming next.
+If you plan a large piece of work, it will be good to produce an outline of the work you need to do and share it with potential contributors.
+Your roadmap should cover your goal and vision and include a timeline for tasks that need to be completed, thus helping new contributors to your project to get an understanding of what is currently happening on the project and what's coming next.
 A roadmap is also a great tool to highlight dependencies among tasks, helping you to schedule work on them efficiently.
 
-Milestones can be really helpful to get to your main goal.
-Milestones can be organised around project goals, dates, events, or timeframes.
+Milestones can be really helpful to reach your main goal.
+They can be organized all around your project goals, dates, events, or timeframes.
```

Comment:
I'm being a bit nitpicky here, but maybe for this line we can go with "They can be organised around your...". I think we're keeping the spelling for British English elsewhere in the book, unless our style guide has changed.

Reply:
Makes sense.